Description: Fixes to man pages.
 Fix NAME sections of the man pages.
 Spell 'separate' correctly.
Author: Andrew Shadura <bugzilla@tut.by>

diff --git a/lib/Net/HTTPServer.pm b/lib/Net/HTTPServer.pm
--- a/lib/Net/HTTPServer.pm
+++ b/lib/Net/HTTPServer.pm
@@ -22,7 +22,7 @@
 
 =head1 NAME
 
-Net::HTTPServer
+Net::HTTPServer - HTTP server
 
 =head1 SYNOPSIS
 
@@ -137,7 +137,7 @@
 =head2 AddServerTokens(token,[token,...])
 
 Adds one or more tokens onto the Server header line that the server sends
-back in a response.  The list is seperated by a ; to distinguish the
+back in a response.  The list is separated by a ; to distinguish the
 various tokens from each other.
 
   $server->AddServerTokens("test/1.3");
diff --git a/lib/Net/HTTPServer/Request.pm b/lib/Net/HTTPServer/Request.pm
--- a/lib/Net/HTTPServer/Request.pm
+++ b/lib/Net/HTTPServer/Request.pm
@@ -22,7 +22,7 @@
 
 =head1 NAME
 
-Net::HTTPServer::Request
+Net::HTTPServer::Request - HTTP request
 
 =head1 SYNOPSIS
 
diff --git a/lib/Net/HTTPServer/Response.pm b/lib/Net/HTTPServer/Response.pm
--- a/lib/Net/HTTPServer/Response.pm
+++ b/lib/Net/HTTPServer/Response.pm
@@ -22,7 +22,7 @@
 
 =head1 NAME
 
-Net::HTTPServer::Response
+Net::HTTPServer::Response - HTTP response
 
 =head1 SYNOPSIS
 
diff --git a/lib/Net/HTTPServer/Session.pm b/lib/Net/HTTPServer/Session.pm
--- a/lib/Net/HTTPServer/Session.pm
+++ b/lib/Net/HTTPServer/Session.pm
@@ -22,7 +22,7 @@
 
 =head1 NAME
 
-Net::HTTPServer::Session
+Net::HTTPServer::Session - HTTP server side client session
 
 =head1 SYNOPSIS
 
